docker-compose.dev.yml

```diff
@@ -81,6 +81,7 @@ services:
             EZS_CONTINUE_DELAY: '60'
             EZS_NSHARDS: '31'
             EZS_CACHE: 'false'
+            EZS_VERBOS: 'true'
```

Comment:
Isn't it `EZS_VERBOSE` (with a final `E`)?

Reply:
I understand why it didn't work ;-)